React DevTools should bump the Electron version to 11.0.1 for darwin-arm64 builds

## Environment:
- macOS 11.0.1 Big Sur, on an Apple Silicon (M1) chip
- React 17.0.1
- React DevTools 4.10.1, running in standalone mode via Electron 11.1.0 (darwin-arm64 build)
- New project using create-react-app . with the useScript hook to add the <script> tag required by React DevTools.

## Steps to reproduce

1. Add React DevTools to a project with `npm install --save react-devtools` (or yarn).
2. react-devtools expects Electron 9.x.x; however builds for darwin-arm64 are only available for 11.1.0. Installation fails with:
```
npm ERR! code 1
npm ERR! path /Users/michael/git/components-blank/node_modules/electron
npm ERR! command failed
npm ERR! command sh -c node install.js
 HTTPError: Response code 404 (Not Found) for https://github.com/electron/electron/releases/download/v9.4.0/electron-v9.4.0-darwin-arm64.zip
 ...
```

Link to code example:  [mglukhovsky/react-devtools-components-blank](https://github.com/mglukhovsky/react-devtools-components-blank)

## The current behavior
Workarounds are [documented here](https://github.com/mglukhovsky/react-devtools-components-blank#electron-1110-required-for-m1); however a 11.1.0 build is required.

## The expected behavior
Bump the version in package.json so this works out of the box.

--- a/packages/react-devtools/package.json
+++ b/packages/react-devtools/package.json
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "cross-spawn": "^5.0.1",
-    "electron": "^9.1.0",
+    "electron": "^11.1.0",
     "ip": "^1.1.4",
     "minimist": "^1.2.3",
     "react-devtools-core": "4.10.1",

